src/display/api.js

```diff
@@ -2068,7 +2084,8 @@ var InternalRenderTask = (function InternalRenderTaskClosure() {
 
       var params = this.params;
       this.gfx = new CanvasGraphics(params.canvasContext, this.commonObjs,
-                                    this.objs, params.imageLayer);
+                                    this.objs, params.imageLayer,
+                                    this.canvasFactory);
```

Comment:
Based on https://github.com/mozilla/pdf.js/pull/8002#discussion_r99020974, please change the order of `params.imageLayer` and `this.canvasFactory` here.